pkg/controller/associated/ios.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package associated
+
+// The iOS format is specified by:
+//   https://developer.apple.com/documentation/safariservices/supporting_associated_domains
+
+import "github.com/google/exposure-notifications-verification-server/pkg/database"
+
+type IosData struct {
```

Comment:
nit: I would expect this to be IOSData given how you're capitalizing it in other places.